tests/test_doctests.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Please avoid modifying the shebang in this way. I discussed this with @mattrose in chat briefly.

Current best practice for Python (e.g. when using virtualenvs) is keeping the term `python` while selecting from `PATH`

In terms for packaging we need to work around at the moment, usually this is handled by standard packaging tools and setuptools.

Until I can improve the integration with setuptools (with 2.0) we could `sed` the shebang in packaging scripts.

Reply:
yes, i revert here.
thx